docs/user_guide/storing/docindex.md

```diff
@@ -37,716 +37,90 @@ Currently, DocArray supports the following vector databases:
-a list of `DocList`s, one for each query, containing the closest matching documents and their similarity scores.
-
-## Filter search and text search
-
-In addition to vector similarity search, the Document Index interface offers methods for text search and filtered search:
-[text_search()][docarray.index.abstract.BaseDocIndex.text_search] and [filter()][docarray.index.abstract.BaseDocIndex.filter],
-as well as their batched versions [text_search_batched()][docarray.index.abstract.BaseDocIndex.text_search_batched] and [filter_batched()][docarray.index.abstract.BaseDocIndex.filter_batched]. [filter_subindex()][docarray.index.abstract.BaseDocIndex.filter_subindex] is for filter on subindex level.
-
-!!! note
-    The [HnswDocumentIndex][docarray.index.backends.hnswlib.HnswDocumentIndex] implementation does not offer support for filter
-    or text search.
-
-    To see how to perform filter or text search, you can check out other backends that offer support.
-
-## Hybrid search through the query builder
-
-Document Index supports atomic operations for vector similarity search, text search and filter search.
-
-To combine these operations into a single, hybrid search query, you can use the query builder that is accessible
-through [build_query()][docarray.index.abstract.BaseDocIndex.build_query]:
-
+### Initialize the Document Index and add data
+Here we initialize an `InMemoryExactNNIndex` instance with the document schema we defined previously, and add the created documents to this index.
 ```python
-# prepare a query
-q_doc = MyDoc(embedding=np.random.rand(128), text='query')
-
-query = (
-    db.build_query()  # get empty query object
-    .find(query=q_doc, search_field='embedding')  # add vector similarity search
-    .filter(filter_query={'text': {'$exists': True}})  # add filter search
-    .build()  # build the query
-)
-
-# execute the combined query and return the results
-results = db.execute_query(query)
-print(f'{results=}')
+doc_index = InMemoryExactNNIndex[MyDoc]()
+doc_index.index(docs)
 ```
 
-In the example above you can see how to form a hybrid query that combines vector similarity search and filtered search
-to obtain a combined set of results.
-
-The kinds of atomic queries that can be combined in this way depends on the backend.
-Some backends can combine text search and vector search, while others can perform filters and vectors search, etc.
-To see what backend can do what, check out the [specific docs](#document-index).
-
-## Access documents by `id`
-
-To retrieve a document from a Document Index, you don't necessarily need to perform a fancy search.
-
-You can also access data by the `id` that was assigned to each document:
-
+### Perform a vector similarity search
+Now, let's perform a similarity search on the document embeddings. 
+As a result, we'll retrieve ten most similar documents and their corresponding similarity scores.
```

Comment:
```suggestion
As a result, we'll retrieve the ten most similar documents and their corresponding similarity scores.
```

Reply:
I thought I removed "the" from every occurance, thx